src/frontend/screens/Achievements/AchievementsLayout/AchievmentStores.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react'
+
+import { StoreRow, Images } from '@hyperplay/ui'
+import { Flex } from '@mantine/core'
+
+import styles from './index.module.css'
+
+import classNames from 'classnames'
+import { observer } from 'mobx-react-lite'
+import AchievementStoreState from 'frontend/state/AchievementStoreState'
+import useSetting from 'frontend/hooks/useSetting'
+import walletState from 'frontend/state/WalletState'
+
+export const AchievementStores = observer(() => {
+  const [steamId] = useSetting('steamId', '')
+
+  const store = AchievementStoreState.store
+  const totalGames = AchievementStoreState.totalGames
+  const setStore = AchievementStoreState.setStore
+  const syncAchievements = AchievementStoreState.syncAchievements
+  const setPlayerStoreId = AchievementStoreState.setPlayerStoreId
+  const getAchievementsStats = AchievementStoreState.getAchievementsStats
```

Comment:
We could consider the same for this as the others suggested. 

```typescript
const { store, totalGames, setStore, syncAchievements, setPlayerStoreId, getAchievementsStats } = AchievementStoreState;```

Reply:
sure thing. updated